privaterelay/templates/includes/dashboard-filter.html

```diff
@@ -4,20 +4,18 @@
 
 <div class="c-filter js-filter-container">
     <div class="c-filter-search">
-        {% if relay_addresses %}
-            <div class="c-filter-icon c-filter-icon-search js-filter-mobile-search-toggle"></div>
-            <form class="js-filter-search-form c-filter-search-form" action="">
-                <input class="c-filter-search-input js-filter-search-input " type="search" name="search" id="search">
-                <div class="c-filter-case-count">
-                    <span class="js-filter-case-visible"></span>
-                    /
-                    <span class="js-filter-case-total"></span>
-                </div>
-                <button class="c-filter-reset js-filter-reset">
-                    <img alt="" src="{% static '/images/x-close.svg' %}"/>
-                </button>
-            </form>
-        {% endif %}
+        <div class="c-filter-icon c-filter-icon-search js-filter-mobile-search-toggle"></div>
+        <form class="js-filter-search-form c-filter-search-form" action="">
+            <input class="c-filter-search-input js-filter-search-input " type="search" name="search" id="search">
+            <div class="c-filter-case-count">
+                <span class="js-filter-case-visible"></span>
+                /
+                <span class="js-filter-case-total"></span>
+            </div>
+            <button class="c-filter-reset js-filter-reset">
```

Comment:
I believe `button` defaults to a submit without an explicit `type="button"` so closing this could inadvertently submit the search form.